Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor

```diff
@@ -0,0 +1,12 @@
+﻿
+<Alert Color="Color.Warning" Visible>
+    📊 Community license has a limitation of @LimitText.
```

Comment:
The message should mention the limitation, but it should also be tailored in a way that promotes our paid tiers.

Maybe something like "The Community License has limitations of X. To unlock full features with our paid tiers—see pricing for details."

Reply:
Now with:
![image](https://github.com/user-attachments/assets/98464dc5-f40a-474d-bf89-460323ba2868)
Shorter and talk less about "money".